### DISCUSSION TITLE
Error: Cannot find module (similar to #6492)

### DISCUSSION THREAD

**Discussion by shanecarlobeticon** [cypress-io/cypress#14337] [Plugins]
How to make this two on(task) in one module?

The first one is,

```
let shouldSkip = false;

module.exports = (on, config) => {
  on('task', {
    resetShouldSkipFlag() {
      shouldSkip = false;
      return null;
    },
    shouldSkip(value) {
      if (value != null) shouldSkip = value;
      return shouldSkip;
    },
    
  })
}; 
```
And the other one is,

```
let percyHealthCheck = require('@percy/cypress/task')

module.exports = (on, config) => {
  on("task", percyHealthCheck);
};
```
I actually tried this one

```
let percyHealthCheck = require('@percy/cypress/task')
let shouldSkip = false;

module.exports = (on, config) => {
  on('task', {
    resetShouldSkipFlag() {
      shouldSkip = false;
      return null;
    },
    shouldSkip(value) {
      if (value != null) shouldSkip = value;
      return shouldSkip;
    },
    
  })
  

on("task", percyHealthCheck);

};
```
That is working on my test runner

but when I am trying to run it in my Browserstack account, I'm getting this error

![image](https://user-images.githubusercontent.com/75348110/103260782-e85be700-49d9-11eb-926e-165d8c551b84.png)

The test will only work in my Browserstack account if I remove the on("task", percyHealthCheck) in my index.js plugin file.
Maybe the way I combined the two tasks in one module is not correct. Can you please help me with that? Thanks!

**Comment by jennifer-shehane** [comment]
You need to have a single on'task' handler in the pluginsFile with each task being comma separated.

```js
let shouldSkip = false;
let percyHealthCheck = require('@percy/cypress/task')

module.exports = (on, config) => {
  on('task', {
    resetShouldSkipFlag() {
      shouldSkip = false;
      return null;
    },
    shouldSkip(value) {
      if (value != null) shouldSkip = value;
      return shouldSkip;
    },
    percyHealthCheck,
  })
}; 
```

**Reply by shanecarlobeticon** [reply]
Thanks @jennifer-shehane, 
but I got this error 

![image](https://user-images.githubusercontent.com/75348110/103329251-eb68dd00-4a96-11eb-9607-e25f6d06a215.png)

**Reply by shanecarlobeticon** [reply]
@jennifer-shehane what do you think is the cause of the error?

**Comment by shanecarlobeticon** [comment]
Hope to find an answer here as our team is now using Cypress.
